templates/generic/digest.html

```diff
@@ -24,51 +26,51 @@
 <h3>{{name}} (+{{newissues.count}}/-{{closedissues.count}}/💬{{issuecommentscount}})</h3>
 {{#newissues.count}}  <p class="new">{{.}} issues created:</p>
   <ul>{{#newissues.list}}
-  <li><a href="{{{payload.issue.html_url}}}">{{{payload.issue.title}}}</a> (by {{actor.display_login}})</li>
+  <li>#{{number}} <a href="{{{payload.issue.html_url}}}">{{{payload.issue.title}}}</a> (by {{actor.display_login}})</li>
   {{/newissues.list}}</ul>
 {{/newissues.count}}
 
-{{#closedissues.count}}  <p>{{.}} issues closed:</p>
-  <ul>{{#closedissues.list}}
-  <li><a href="{{{payload.issue.html_url}}}">{{{payload.issue.title}}}</a></li>
-  {{/closedissues.list}}</ul>
-{{/closedissues.count}}
-
 {{#commentedissues.count}}  <p>{{.}} issues received {{issuecommentscount}} new comments:</p>
   <ul>{{#commentedissues.list}}
   <li>#{{number}} <a href="{{{url}}}">{{{title}}}</a> ({{commentscount}} by {{#commentors}}{{name}}{{^last}}, {{/last}}{{/commentors}})</li>
   {{/commentedissues.list}}</ul>
 {{/commentedissues.count}}
 
+{{#closedissues.count}}  <p>{{.}} issues closed:</p>
+  <ul>{{#closedissues.list}}
+  <li>#{{number}} <a href="{{{payload.issue.html_url}}}">{{{payload.issue.title}}}</a></li>
+  {{/closedissues.list}}</ul>
+{{/closedissues.count}}
+
 {{/activeissuerepos}}{{/activeissues}}
 
 {{#activeprs}}<h2>Pull requests</h2>
 {{#activeprrepos}}
 <h3>{{name}} (+{{newpr.count}}/-{{mergedpr.count}}/💬{{prcommentscount}})</h3>
 {{#newpr.count}}  <p class="new">{{.}} pull requests submitted:</p>
   <ul>{{#newpr.list}}
-  <li><a href="{{{payload.pull_request.html_url}}}">{{{payload.pull_request.title}}}</a> (by {{actor.display_login}})</li>
+  <li>#{{number}} <a href="{{{payload.pull_request.html_url}}}">{{{payload.pull_request.title}}}</a> (by {{actor.display_login}})</li>
   {{/newpr.list}}</ul>
 {{/newpr.count}}
 
-{{#mergedpr.count}}  <p>{{.}} pull requests merged:</p>
-  <ul>{{#mergedpr.list}}
-  <li><a href="{{{payload.pull_request.html_url}}}">{{{payload.pull_request.title}}}</a></li>
-  {{/mergedpr.list}}</ul>
-
 {{/mergedpr.count}}
```

Comment:
this needs to move with the rest of the  `{{#mergedpr.count}}` section (and this explains why the Travis CI build fails, although I suspect it will fail for other reasons after that gets fixed)